### DISCUSSION TITLE
Add Support for Downloading charts from widgets

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#1160] [Ideas]
i have Generated chart in dashboard like this

![image](https://user-images.githubusercontent.com/30294553/148514025-1d575ee8-e9e5-42e8-94a2-70a9f4c8af14.png)

It would be nice to have download button on top of the chart to download it

**Comment by danharrin** [comment]
This is something you can implement yourself with a custom widget, but probably not anything we're going to submit to the core any time soon
